apisix/plugins/log-rotate.lua

```diff
@@ -76,7 +77,6 @@ end
 
 
 local function get_log_path_info(file_type)
-    local_conf = core.config.local_conf()
```

Comment:
why remove this line?

Reply:
`local_conf` is initialized int the `init` function now.